### PR TITLE
Create only one nifti

### PR DESCRIPTION
**Pull request by myigitavci**
To avoid creating multiple nifti files for single series, I added -m flag. Explanation:
  -m : merge 2D slices from same series regardless of echo, exposure, etc. (n/y or 0/1/2, default 2) [no, yes, auto]

### CODE REVIEW

**Comment by yigitsoy** [comment]
Can you make ignore and merge flags as class variables of converter and pass them to run_bin function? This way signatures become simpler and less changes are needed.

**Comment by myigitavci** [comment]
@yigitsoy can you take a look again

File: nekton/utils/bin.py [hunk 1]
```diff
@@ -18,21 +19,21 @@ def make_exec_bin():
     process.communicate()
 
 
-def run_bin(path: str, outpath:str = None):
+def run_bin(path: str, outpath:str = None,ignore_flag:str = None,merge_flag:str = None):
```
Comment (yigitsoy) [inline nekton/utils/bin.py]: Please set the default string values here instead of None

Comment (myigitavci) [reply nekton/utils/bin.py]: done

**Review by yigitsoy — APPROVED** [review]

**Pull request merged**